include/boost/geometry/strategies/geographic/distance_cross_track.hpp

```diff
@@ -697,10 +697,13 @@ class geographic_cross_track
         point p2 = point(lon2, lat2);
         point p3 = point(lon3, lat3);
 
-        geometry::strategy::distance::cross_track<CT> cross_track(earth_radius);
+        using haversine_t = geometry::strategy::distance::haversine<CT>;
+        using cross_track_t = geometry::strategy::distance::cross_track<void, haversine_t>;
+
+        cross_track_t cross_track(earth_radius);
         CT s34_sph = cross_track.apply(p3, p1, p2);
 
-        geometry::strategy::distance::haversine<CT> str(earth_radius);
+        haversine_t str(earth_radius);
```

Comment:
They probably can be made `const`. But also that was already before.

Reply:
Right, I automatically changed the types. I added const now, also in other parts of the file.